src/skmatter/_selection.py

```diff
@@ -84,7 +78,8 @@ class GreedySelector(SelectorMixin, MetaEstimatorMixin, BaseEstimator):
     X_selected_ : ndarray,
                   Matrix containing the selected samples or features, for use in fitting
     y_selected_ : ndarray,
-                  In sample selection, the matrix containing the selected targets, for use in fitting
+                  In sample selection, the matrix containing the selected targets, for
+                  use in fitting
```

Comment:
There are small, unnecessary formatting changes throughout here.

Reply:
Why unnecessary? There is a line width of 88 characters. This should be obeyed to please the linter.